packages/desktopjs-electron/tests/electron.spec.ts

```diff
@@ -729,6 +742,22 @@ describe("ElectronContainer", () => {
                 expect(layout.windows[0].name === "win1")
             }).then(done);
         });
+
+        it("setOptions allows the auto startup settings to be turned on", () => {
+            spyOn(electron, "setLoginItemSettings").and.callThrough();
+            container.setOptions({ autoStartOnLogin: true });
+            expect(electron.setLoginItemSettings).toHaveBeenCalledWith({ openAtLogin: true });
+        });
+
+        it("setOptions errors out on setLoginItemSettings", () => {
+            spyOn(electron, "setLoginItemSettings").and.callFake(() => {
+                throw new Error("something went wrong");
+            });
+            spyOn(console, "error").and.callThrough();
+            container.setOptions({ autoStartOnLogin: true });
+            expect(electron.setLoginItemSettings).toHaveBeenCalledWith({ openAtLogin: true });
+            expect(console.error).toHaveBeenCalled();
```

Comment:
Codacy found an issue: [Unexpected console statement.](https://app.codacy.com/manual/brian.ingenito/desktopJS/pullRequest?prid=5916792)

Reply:
its the error condition which has been tested. I will have to disable the check on this line to by pass it.